benchmark/README.md

```diff
@@ -26,10 +26,10 @@ $ cargo build -p benchmark --release
 
 ```
 $ cd ../grpc
-$ python2.7 tools/run_tests/run_performance_tests.py -l rust
+$ python3 tools/run_tests/run_performance_tests.py -l rust
 ```
 
-Checkout `python2.7 tools/run_tests/run_performance_tests.py --help` to see custom options.
+Checkout `python3 tools/run_tests/run_performance_tests.py --help` to see custom options.
```

Comment:
The reason for the change?

Reply:
grpc has upgraded to python3. See https://github.com/grpc/grpc/pull/27135.